Resources/Prototypes/Entities/Buildings/vending_machines.yml

```diff
@@ -120,7 +120,7 @@
 - type: entity
   parent: VendingMachine
   id: VendingMachineCola
-  name: Robust Softdrinks
+  name: Robust softdrinks
```

Comment:
Wouldn't softdrinks still be part of the brand name in this case? And many other ones down below.

Reply:
Done